### PR TITLE
ReadMe Fix - Remove config info, add config settings desc

### PR DESCRIPTION
**Pull request by ohare93**
Fixed, as discussed :+1: :sweat_smile:

### CODE REVIEW

**Comment by Stvad** [comment]
there's also almost the same thing in https://github.com/Stvad/CrowdAnki/blob/master/misc/ankiweb_description.html (I know duplication, bad. But Ankiweb is weierd - you can't just post the rendered version of markdown page there 😞  )

File: README.md [hunk 1]
```diff
@@ -167,9 +167,19 @@ So, for example, to get my [git deck](https://github.com/Stvad/Software_Engineer
...
+You can take snapshots manually via `File > CrowdAnki: Snapshot` menu action. 
+Or you can enable automated snapshots in add-on configuration (see below) which will take a snapshot each time you open or close anki.
+
+## Configuration Settings
+You can find the add-on's config page via `Tools -> Add-ons, select CrowdAnki then hit Config`. 
+In here you can set a some config options, as well as a detailed explanation on what each does, with examples and default behaviour.
+
+Here is a general summary of the currently available config options:
+ - `Snapshot Path`: location of the snapshot will be exported to
+ - `Snapshot Root Decks`: what decks should be included in the snapshot
```
Comment (Stvad) [inline README.md]: 's not quite that (https://github.com/Stvad/CrowdAnki/blob/585f570b3ebcd06ee3500caf661290f95cc0943e/crowd_anki/config.md)

File: README.md [hunk 2]
```diff
@@ -167,9 +167,19 @@ So, for example, to get my [git deck](https://github.com/Stvad/Software_Engineer
 ## Snapshots
 
 **CrowdAnki** can help you preserve **the history of edits for your decks**.  
-It does this by exporting them in a specified location and creating a git commit each time you do a snapshot.
-
-You can take snapshots manually via `File > CrowdAnki: Snapshot` menu action.  
-Or you can enable automated snapshots in plugin configuration. You can find more details on how to do it [here](crowd_anki/config.md) 
-
-Other things you can [configure](crowd_anki/config.md) is the location of the snapshot and what decks should be included into it. 
+It does this by exporting them in a specified location and creating a git commit each time you do a snapshot.
```
Comment (Stvad) [inline README.md]: why is this updateted? are you introducing windows line endings? 🧐

Comment (ohare93) [reply README.md]: This was just the deletion of lines 171-175. Line 170 is unchanged :thinking: silly Github (also I do all my dev on Linux fyi :wink:)

File: README.md (hunk 1 above)
Comment (ohare93) [reply README.md]: Fixed :+1:

**Comment by ohare93** [comment]
> there's also almost the same thing in https://github.com/Stvad/CrowdAnki/blob/master/misc/ankiweb_description.html (I know duplication, bad. But Ankiweb is weierd - you can't just post the rendered version of markdown page there  )

:roll_eyes: :disappointed: 

I was confused when I looked at this, as your config here is point to `file:///Users/sitalov/Dropbox/SoftwareEngineering/Projects/Anki/CrowdAnki/crowd_anki/config.md` :sweat_smile: yet there is no such link in the [AnkiWeb page](https://ankiweb.net/shared/info/1788670778)'s for the add-on :thinking: I presume AnkiWeb removes all `file:///` links, and therefore at least this part was never broken? :sweat_smile:

**Pull request merged**